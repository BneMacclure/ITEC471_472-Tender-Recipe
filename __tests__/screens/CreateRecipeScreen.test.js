import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import CreateRecipeScreen from '../../screens/CreateRecipeScreen';


describe('<CreateRecipeScreen />', () => {
    it('CreateRecipeScreen snapshot test' , () => {
        const tree = renderer.create(<CreateRecipeScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})