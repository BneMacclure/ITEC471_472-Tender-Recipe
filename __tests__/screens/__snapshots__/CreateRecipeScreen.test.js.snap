// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateRecipeScreen /> CreateRecipeScreen snapshot test 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "height": 222.33333333333334,
        "width": 750,
      }
    }
  >
    <Image
      resizeMode="contain"
      source={1}
      style={
        Object {
          "alignSelf": "center",
          "height": "50%",
          "marginTop": 15,
          "width": "50%",
        }
      }
    />
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#121212",
          "fontSize": 20,
          "marginTop": "5%",
        }
      }
    >
      What's your recipe?
    </Text>
  </View>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "justifyContent": "space-between",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "width": 750,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignSelf": "center",
            "height": "100%",
            "marginTop": 35,
            "width": 292,
          }
        }
      >
        <View
          style={
            Object {
              "height": 604,
              "marginLeft": -129,
              "marginTop": -37,
              "width": 591,
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "height": 592,
                "left": 0,
                "top": 12,
                "width": 591,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 592,
                    "width": 591,
                  },
                  Object {},
                ]
              }
            />
            <View
              style={
                Object {
                  "backgroundColor": "rgba(255,255,255,1)",
                  "borderColor": "#000000",
                  "borderRadius": 100,
                  "borderWidth": 1,
                  "height": 24,
                  "left": 220,
                  "position": "absolute",
                  "top": -10,
                  "width": 110,
                }
              }
            >
              <Text
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "rgba(222,5,5,1)",
                    "fontSize": 12,
                    "marginTop": 2,
                  }
                }
              >
                FILL IN BELOW
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 1,
                  "marginBottom": 527,
                  "marginLeft": 166.66666666666666,
                  "marginRight": 170,
                  "marginTop": 20,
                }
              }
            >
              <View
                style={
                  Object {
                    "height": 61,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "rgba(255,255,255,1)",
                      "fontSize": 20,
                      "fontWeight": "bold",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                    }
                  }
                >
                  Recipe Name
                </Text>
                <TextInput
                  allowFontScaling={true}
                  enablesReturnKeyAutomatically={true}
                  multiline={false}
                  onChangeText={[Function]}
                  placeholder=""
                  rejectResponderTermination={true}
                  style={
                    Object {
                      "backgroundColor": "rgba(255,255,255,1)",
                      "borderColor": "#000000",
                      "borderRadius": 10,
                      "borderWidth": 1,
                      "color": "#121212",
                      "fontSize": 16,
                      "height": 40,
                      "marginTop": 30,
                      "paddingLeft": 8,
                      "width": 681.8181818181818,
                    }
                  }
                  underlineColorAndroid="transparent"
                />
              </View>
              <Text
                style={
                  Object {
                    "color": "rgba(255,255,255,1)",
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "marginLeft": 1,
                    "marginTop": 15,
                  }
                }
              >
                The Ingredients
              </Text>
              <TextInput
                allowFontScaling={true}
                enablesReturnKeyAutomatically={true}
                multiline={true}
                onChangeText={[Function]}
                placeholder=""
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "rgba(255,255,255,1)",
                    "borderColor": "#000000",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "color": "#121212",
                    "fontSize": 16,
                    "height": 204,
                    "paddingLeft": 8,
                    "paddingTop": 2,
                    "textAlignVertical": "top",
                    "width": 681.8181818181818,
                  }
                }
                underlineColorAndroid="transparent"
              />
              <Text
                style={
                  Object {
                    "color": "rgba(255,255,255,1)",
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "marginLeft": 1,
                    "marginTop": 8,
                  }
                }
              >
                The Instructions
              </Text>
              <TextInput
                allowFontScaling={true}
                enablesReturnKeyAutomatically={true}
                multiline={true}
                onChangeText={[Function]}
                placeholder=""
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "rgba(255,255,255,1)",
                    "borderColor": "#000000",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "color": "#121212",
                    "fontSize": 16,
                    "height": 204,
                    "paddingLeft": 8,
                    "paddingTop": 2,
                    "textAlignVertical": "top",
                    "width": 681.8181818181818,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 760,
              "width": 292,
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "height": 715,
                "marginLeft": -135,
                "marginTop": -24,
                "width": 591,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 715,
                    "width": 591,
                  },
                  undefined,
                ]
              }
            />
            <View
              style={
                Object {
                  "alignSelf": "center",
                  "backgroundColor": "rgba(255,255,255,1)",
                  "borderColor": "#000000",
                  "borderRadius": 100,
                  "borderWidth": 1,
                  "height": 24,
                  "marginRight": 20,
                  "marginTop": 87,
                  "width": 220,
                }
              }
            >
              <Text
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "rgba(222,5,5,1)",
                    "fontSize": 12,
                    "marginTop": 2,
                  }
                }
              >
                SELECT ALLERGENS THAT APPLY
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 1,
                  "height": 43,
                  "marginLeft": 163.0434782608696,
                  "marginTop": 15,
                  "width": 707.5471698113207,
                }
              }
            >
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Gluten
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Nuts
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Shellfish
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Dairy
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Eggs
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Fish
                  </Text>
                </View>
              </View>
              <View
                accessibilityRole="checkbox"
                accessibilityState={
                  Object {
                    "checked": false,
                  }
                }
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderColor": "#ededed",
                    "borderRadius": 10,
                    "borderWidth": 1,
                    "margin": 5,
                    "marginLeft": 10,
                    "marginRight": 10,
                    "opacity": 1,
                    "padding": 10,
                  }
                }
                testID="checkbox"
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "rgba(111,111,111,1)",
                        "fontSize": 20,
                        "fontWeight": "bold",
                        "marginLeft": 10,
                        "marginRight": 10,
                        "paddingBottom": 4,
                      }
                    }
                    testID="checkboxTitle"
                  >
                    Soy
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "alignSelf": "center",
              "height": 890,
              "marginTop": -10,
              "width": 292,
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "height": 740,
                "marginLeft": -135,
                "marginTop": -163,
                "width": 870,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 740,
                    "width": 870,
                  },
                  Object {},
                ]
              }
            />
            <View
              style={
                Object {
                  "backgroundColor": "rgba(255,255,255,1)",
                  "borderColor": "#000000",
                  "borderRadius": 100,
                  "borderWidth": 1,
                  "height": 24,
                  "marginLeft": 208,
                  "marginTop": 70,
                  "width": 147,
                }
              }
            >
              <Text
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "rgba(222,5,5,1)",
                    "fontSize": 12,
                    "marginTop": 2,
                  }
                }
              >
                UPLOAD A PICTURE!
              </Text>
            </View>
            <View
              style={
                Object {
                  "backgroundColor": "rgba(255,255,255,1)",
                  "borderRadius": 10,
                  "height": 514,
                  "marginLeft": 110,
                  "marginTop": 20,
                  "width": 625,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "rgba(239,115,15,1)",
                    "borderRadius": 100,
                    "height": 74,
                    "justifyContent": "center",
                    "marginTop": 35,
                    "width": 74,
                  }
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "#121212",
                    "fontSize": 18,
                    "marginTop": 10,
                  }
                }
              >
                Tap the button Below
              </Text>
              <Text
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "#121212",
                    "marginTop": 1,
                  }
                }
              >
                See your photo here!
              </Text>
              <View
                style={
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "#E6E6E6",
                    "borderColor": "#000000",
                    "borderRadius": 1,
                    "borderStyle": "dashed",
                    "borderWidth": 1,
                    "height": 245,
                    "marginTop": 9,
                    "width": 165,
                  }
                }
              />
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "rgba(249,71,35,1)",
                    "borderColor": "#000000",
                    "borderRadius": 10,
                    "borderStyle": "solid",
                    "borderWidth": 1,
                    "height": 42,
                    "justifyContent": "center",
                    "marginTop": 15,
                    "opacity": 1,
                    "width": 198,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "color": "rgba(255,255,255,1)",
                      "fontSize": 18,
                    }
                  }
                >
                  BROWSE PHOTOS
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "alignSelf": "center",
              "height": 96,
              "marginTop": -360,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "backgroundColor": "rgba(249,71,35,1)",
                "borderRadius": 10,
                "height": 56,
                "justifyContent": "center",
                "opacity": 1,
                "width": 681.8181818181818,
              }
            }
          >
            <Text
              style={
                Object {
                  "alignSelf": "center",
                  "color": "rgba(255,255,255,1)",
                  "fontWeight": "bold",
                }
              }
            >
              Submit
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
